documentation/asciidoc/topics/proc_graceful_shutdown.adoc

```diff
@@ -0,0 +1,12 @@
+[id='shutting_down-{context}']
+= Shutting Down {brandname} Clusters
+Gracefully stop all {brandname} pods in the cluster so that permanent cache definitions and persisted data is available the next time pods start.
+
```

Comment:
MORE INFO:
The operator makes sure that the capacity of the PV volume mounted on the pod is enough for storing all the data during the shutdown procedure. The user must take care of sizing the PV, if he explicitly specify a PV size (DataGrid only).

Reply:
if the capacity of the PV is less than the size of the cache what happens? data loss or does the operator not complete the shutdown unless all data is persisted? so the shutdown hangs if the PV is too small.

Reply:
I think you get a no space left on device. iirc @ryanemerson implemented that part and maybe can help us.

Reply:
I think it was Tristan who implemented graceful shutdown, but I assume the same as you @rigazilla that an exception will just be thrown. Kubernetes now supports dynamic expansion of persistent volumes, so that's a potential workaround https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/.